src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private Ball ball = new Ball();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
```

Comment:
```suggestion
        ball.setNumber(random.nextInt(MAX_NUMBER));
```
avoid magic numbers - use constants